test/helpers/kubectl.go

```diff
@@ -4467,7 +4467,7 @@ func (kub *Kubectl) ensureKubectlVersion() error {
 	}
 
 	versionstring := fmt.Sprintf("%s.%s", v.ClientVersion.Major, v.ClientVersion.Minor)
-	if versionstring == GetCurrentK8SEnv() {
+	if strings.HasPrefix(GetCurrentK8SEnv(), versionstring) {
```

Comment:
I think the two arguments are inverted, no? It's going to run `strings.HasPrefix("1.20", "1.20+")` where the second argument is expected to be the prefix.

Also, shouldn't we instead strip the plus sign? Checking the prefix won't work if two versions are ever legitimately the prefix of one another, e.g., 1.200 and 1.20. Sure, it seems unlikely today, but if it's the same, we might as well make the helper function correct for all inputs.